jena-geosparql/pom.xml

```diff
@@ -98,10 +147,12 @@
 
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-slf4j2-impl</artifactId>
+      <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
`log4j-slf4j2-impl` is for slf4j v2.x.x, `log4j-slf4j-impl` is for slf4j v1.x.x

Jena uses slf4j v2.x.x